include/tcpdf_fonts.php

```diff
@@ -2003,7 +2003,10 @@ public static function UTF8StringToArray($str, $isunicode=true, &$currentfont) {
 			$chars = str_split($str);
 			$carr = array_map('ord', $chars);
 		}
-		$currentfont['subsetchars'] += array_fill_keys($carr, true);
+		if (is_array($currentfont['subsetchars']) && is_array($carr))
+			$currentfont['subsetchars'] += array_fill_keys($carr, true);
+		else
+			array_merge($currentfont['subsetchars'], $carr);
```

Comment:
the resulting array need to be assigned?

Reply:
Right, I thought array_merge does that automatically. Added a new commit with assignment.